openupgrade_scripts/scripts/base/15.0.1.3/pre-migration.py

```diff
@@ -47,6 +47,28 @@
     ("l10n_ec.state_ec_24", "base.state_ec_24"),
 ]
 
+
+def fill_point_of_sale_refunded_orderline_id_column(cr):
+    # fill pos_order_line field
```

Comment:
Check if the module is installed. e. g.: https://github.com/oca/openupgrade/blob/15.0/openupgrade_scripts/scripts/website_crm/15.0.2.1/pre-migration.py#L6-L9

Or checking if the origin columns exist.